templates/taxi/form/car_confirm_delete.html

```diff
@@ -0,0 +1,14 @@
+{% extends "base.html" %}
+{% load crispy_forms_filters %}
+
+{% block content %}
+  <p>Are you sure you want to delete the car: {{ car.model }}</p>
+  <p>All car with this manufacturer will be also deleted</p>
+
+  <form action="" method="POST">
+    {% csrf_token %}
+    {{ form|crispy }}
+
+    <input type="submit" value="Yes, delete" class="btn btn-danger">
+  </form>
+{% endblock %}
```

Comment:
Please add empty lines to files where missing. Then github won't show these red circles in the end of the files